blocks/library/cover-image/index.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * WordPress dependencies
+ */
+import Button from 'components/button';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query } from '../../api';
+import Editable from '../../editable';
+// TODO: Revisit when we have a common components solution
+import Dashicon from '../../../components/dashicon';
```

Comment:
Should we move this to the WordPress dependencies `import Dashicon from 'components/dashicon';`?